mod-vendors/schemas/vendor.json

```diff
@@ -167,7 +167,167 @@
+            "username": {
+              "description": "The login username for this EDI",
+              "type": "string"
+            },
+            "password": {
+              "description": "The login password for this EDI",
+              "type": "string"
+            },
+            "ftp_mode": {
+              "description": "The FTP mode for this EDI",
+              "type": "string"
+            },
+            "ftp_conn_mode": {
+              "description": "The FTP connection mode for this EDI",
+              "type": "string"
+            },
+            "ftp_port": {
+              "description": "The port for this EDI",
+              "type": "string"
+            },
+            "order_directory": {
+              "description": "The order directory for this EDI",
+              "type": "string"
+            },
+            "invoice_directory": {
+              "description": "The invoice directory for this EDI",
+              "type": "string"
+            },
+            "notes": {
+              "description": "The notes for this EDI",
+              "type": "string"
+            }
+          }
+        },
+        "edi_job": {
+          "description": "The job object for this EDI",
+          "type": "object",
+          "properties": {
+            "schedule_edi": {
+              "description": "The schedule for this EDI job",
+              "type": "boolean"
+            },
+            "date": {
+              "description": "The date for this EDI job",
+              "type": ["string", "null"],
+              "format": "date-time"
+            },
+            "time": {
+              "description": "The time for this EDI job",
+              "type": ["string", "null"],
+              "format": "date-time"
+            },
+            "is_monday": {
+              "description": "The setting to run this EDI job on Mondays",
+              "type": "boolean"
+            },
+            "is_tuesday": {
+              "description": "The setting to run this EDI job on Tuesdays",
+              "type": "boolean"
+            },
+            "is_wednesday": {
+              "description": "The setting to run this EDI job on Wednesdays",
+              "type": "boolean"
+            },
+            "is_thursday": {
+              "description": "The setting to run this EDI job on Thursdays",
+              "type": "boolean"
+            },
+            "is_friday": {
+              "description": "The setting to run this EDI job on Fridays",
+              "type": "boolean"
+            },
+            "is_saturday": {
+              "description": "The setting to run this EDI job on Saturdays",
+              "type": "boolean"
+            },
+            "is_sunday": {
+              "description": "The setting to run this EDI job on Sundays",
+              "type": "boolean"
+            },
+            "send_to_emails": {
+              "description": "The email addresses to notify when this EDI job runs",
```

Comment:
Addresses (plural)... so a delimited list?  If so, what's the delimeter?  comma?  semi-colon? space?

Reply:
Added that it is a comma delimited list in the property description.